common/src/services/search.service.ts

```diff
@@ -294,12 +295,14 @@ export class SearchService implements SearchServiceAbstraction {
     const checkFields = fields.every((i: any) => searchableFields.includes(i));
 
     if (checkFields) {
-      return token
-        .toString()
-        .normalize("NFD")
-        .replace(/[\u0300-\u036f]/g, "");
+      return this.normalizeSearchQuery(token.toString());
     }
 
     return token;
   }
+
+  //Normalize diacritics characters from text
```

Comment:
```suggestion
  // Remove accents/diacritics characters from text. This regex is equivalent to the Diacritic unicode property escape, i.e. it will match all diacritic characters.
```

Reply:
I wanted to explain the regex in particular